frontend/src/scenes/saved-insights/savedInsightsLogic.ts

```diff
@@ -255,6 +255,7 @@ export const savedInsightsLogic = kea<savedInsightsLogicType>({
         },
         [dashboardsModel.actionTypes.updateDashboardInsight]: () => actions.loadInsights(),
         [dashboardsModel.actionTypes.duplicateDashboardSuccess]: () => actions.loadInsights(),
+        [insightsModel.actionTypes.insightsDeletedWithDashboard]: () => actions.loadInsights(),
```

Comment:
Perhaps we can even omit the `insightsDeletedWithDashboard` action and listen directly for `deleteDashboardLogic.actionTypes.submitDeleteDashboardSuccess`?

Reply:
👩‍🍳👌